62_CAD/common.hlsl

```diff
@@ -1,11 +1,30 @@
 #ifndef _CAD_EXAMPLE_COMMON_HLSL_INCLUDED_
 #define _CAD_EXAMPLE_COMMON_HLSL_INCLUDED_
 
+#include <nbl/builtin/hlsl/portable/float64_t.hlsl>
+#include <nbl/builtin/hlsl/portable/vector_t.hlsl>
+#include <nbl/builtin/hlsl/portable/matrix_t.hlsl>
 #include <nbl/builtin/hlsl/limits.hlsl>
 #include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
 #include <nbl/builtin/hlsl/shapes/beziers.hlsl>
 #ifdef __HLSL_VERSION
 #include <nbl/builtin/hlsl/math/equations/quadratic.hlsl>
+#include <nbl/builtin/hlsl/jit/device_capabilities.hlsl>
+#endif
+
+using namespace nbl::hlsl;
+
+// because we can't use jit/device_capabilities.hlsl in c++ code
+#ifdef __HLSL_VERSION
+using cpp_hlsl_portable_float64_t = portable_float64_t<jit::device_capabilities>;
```

Comment:
hmm, you're redoing this again with different names in vertex_shader
choose a single name for it and use the same in VertexShader.
`PortableFloat64` instead of `ActualFloat64` seems like a good choice. also the Pascal Case naming indicates it might actually be a struct/class emulating it